ui-modules/home/app/views/main/deploy/modal.template.html

```diff
@@ -17,4 +17,4 @@
   under the License.
 -->
 <br-svg type="close" class="modal-close pull-right" ng-click="$dismiss('Dismiss deployment modal')"></br-svg>
-<brooklyn-quick-launch app="app" args="args" locations="locations" callback="$close"></brooklyn-quick-launch>
+<brooklyn-quick-launch app="app" args="args" locations="locations" simpleMode="$scope.usingCatalogQuickLaunchTags" callback="$close"></brooklyn-quick-launch>
```

Comment:
Should call directly `usingCatalogQuickLaunchTags` as the current context is `$scope`

Reply:
Also, `simpleMode` is not used anywhere, not is a option for the directive `brooklyn-quick-launch`. What is it for exactly?

Reply:
quite right, it's not used.  it was to suppress "Create location" when using quick-launch tags (as discussed below).  removed.